// node-capmonster/dist/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.TurnstileTask = exports.TenDITask = exports.RecaptchaV3Task = exports.RecaptchaV2EnterpriseTask = exports.RecaptchaV2Task = exports.ImageToTextTask = exports.HCaptchaTask = exports.GeeTestV4Task = exports.GeeTestTask = exports.FuncaptchaTask = exports.DataDomeTask = exports.ComplexImageTask = exports.AmazonTask = exports.CapmonsterError = void 0;
var capmonster_error_1 = require("./capmonster_error");
Object.defineProperty(exports, "CapmonsterError", { enumerable: true, get: function () { return capmonster_error_1.CapmonsterError; } });
var aws_waf_1 = require("./tasks/aws_waf");
Object.defineProperty(exports, "AmazonTask", { enumerable: true, get: function () { return aws_waf_1.AmazonTask; } });
var complex_image_1 = require("./tasks/complex_image");
Object.defineProperty(exports, "ComplexImageTask", { enumerable: true, get: function () { return complex_image_1.ComplexImageTask; } });
var data_dome_1 = require("./tasks/data_dome");
Object.defineProperty(exports, "DataDomeTask", { enumerable: true, get: function () { return data_dome_1.DataDomeTask; } });
var fun_captcha_1 = require("./tasks/fun_captcha");
Object.defineProperty(exports, "FuncaptchaTask", { enumerable: true, get: function () { return fun_captcha_1.FuncaptchaTask; } });
var geetest_1 = require("./tasks/geetest");
Object.defineProperty(exports, "GeeTestTask", { enumerable: true, get: function () { return geetest_1.GeeTestTask; } });
var geetest_v4_1 = require("./tasks/geetest_v4");
Object.defineProperty(exports, "GeeTestV4Task", { enumerable: true, get: function () { return geetest_v4_1.GeeTestV4Task; } });
var hcaptcha_1 = require("./tasks/hcaptcha");
Object.defineProperty(exports, "HCaptchaTask", { enumerable: true, get: function () { return hcaptcha_1.HCaptchaTask; } });
var image_to_text_1 = require("./tasks/image_to_text");
Object.defineProperty(exports, "ImageToTextTask", { enumerable: true, get: function () { return image_to_text_1.ImageToTextTask; } });
var recaptcha_v2_1 = require("./tasks/recaptcha_v2");
Object.defineProperty(exports, "RecaptchaV2Task", { enumerable: true, get: function () { return recaptcha_v2_1.RecaptchaV2Task; } });
var recaptcha_v2_enterprise_1 = require("./tasks/recaptcha_v2_enterprise");
Object.defineProperty(exports, "RecaptchaV2EnterpriseTask", { enumerable: true, get: function () { return recaptcha_v2_enterprise_1.RecaptchaV2EnterpriseTask; } });
var recaptcha_v3_1 = require("./tasks/recaptcha_v3");
Object.defineProperty(exports, "RecaptchaV3Task", { enumerable: true, get: function () { return recaptcha_v3_1.RecaptchaV3Task; } });
var tendi_1 = require("./tasks/tendi");
Object.defineProperty(exports, "TenDITask", { enumerable: true, get: function () { return tendi_1.TenDITask; } });
var turnstile_1 = require("./tasks/turnstile");
Object.defineProperty(exports, "TurnstileTask", { enumerable: true, get: function () { return turnstile_1.TurnstileTask; } });
